tools/gen_locale.dart

```diff
@@ -0,0 +1,58 @@
+import 'dart:async';
```

Comment:
A comment that explains what this app is for (even though we may not run it very often) would be good.

Reply:
done